backend/pom.xml

```diff
@@ -40,10 +39,12 @@
     </dependencies>
 
     <build>
+        <finalName>musicompass</finalName>
         <plugins>
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
+                <version>2.4.3</version>
```

Comment:
any specific reason you included a specific version here? should not be necessary

Reply:
spring-boot-mavin-plugin was first marked in red and specifying the version resolved this issue - found it here https://stackoverflow.com/questions/63468269/maven-plugin-not-found-in-intellij-ide